packages/backend/src/routes/courses.ts

```diff
@@ -25,7 +25,7 @@ coursesRouter.get("/:id", async (ctx, next) => {
     const { id } = ctx.params;
 
     try {
-        const rows = await new Promise((resolve, reject) => {
+        const course = await new Promise((resolve, reject) => {
             db.get(
                 `SELECT * FROM courses WHERE id = ${id}`,
                 [],
```

Comment:
I don't think we should pass the id variable directly to the string (injection vulnerability). Perhaps:

```ts
     const course = await new Promise((resolve, reject) => {
            db.get(
                "SELECT * FROM courses WHERE id = ?",
                [id],
                (err, rows) => {
                    if (err) {
                        reject(err);
                    } else {
                        resolve(rows);
                    }
                },
            );
        });
```